src/js/components/AlertDialogComponent.jsx

```diff
@@ -0,0 +1,72 @@
+var React = require("react/addons");
+var classNames = require("classnames");
+
+var Util = require("../helpers/Util");
+var ModalComponent = require("../components/ModalComponent");
+
+var AlertModalComponent = React.createClass({
+  displayName: "AlertModalComponent",
+
+  propTypes: {
+    data: React.PropTypes.shape({
+      actionButtonLabel: React.PropTypes.string.isRequired,
+      message: React.PropTypes.string.isRequired,
+      state: React.PropTypes.string.isRequired,
```

Comment:
It took me a while to figure out what ```state``` or ```level``` should mean.
```severity``` has a real meaning, also without context. Let's use that please.